packages/mco/components/disaster-recovery/app-failover-relocate/error-messages.tsx

```diff
@@ -0,0 +1,202 @@
+        </p>
+      </Trans>
+    ),
+    variant: AlertVariant.danger,
+  },
+  [ErrorMessageType.MANAGED_CLUSTERS_ARE_DOWN]: {
+    title: t('1 or more managed clusters are offline.'),
+    message: (
+      <Trans t={t}>
+        <ul>
+          <li>
+            The status for both the primary and target clusters must be
+            available for relocating. Check the status and try again.
+          </li>
+          <li>
+            To bring the cluster online, refer to the instructions in the
+            documentation
+            <ViewDocumentation
+              doclink={DOC_LINKS.ACM_OFFLINE_CLUSTER}
+              text="Troubleshoot"
+            />
+          </li>
+        </ul>
+      </Trans>
+    ),
+    variant: AlertVariant.danger,
+  },
+  [ErrorMessageType.TARGET_CLUSTER_IS_NOT_AVAILABLE]: {
+    title: t('Target cluster is offline.'),
+    message: (
+      <Trans t={t}>
+        <p>
+          To begin failover, the target cluster must be available. Check the
+          status and try again. If the managed cluster status is offline, follow
+          the instructions in the documentation
+          <ViewDocumentation
+            doclink={DOC_LINKS.ACM_OFFLINE_CLUSTER}
+            text="Troubleshoot"
+          />
+        </p>
+      </Trans>
+    ),
+    variant: AlertVariant.danger,
+  },
+  [ErrorMessageType.SOME_CLUSTERS_ARE_FENCED]: {
+    title: t('Some clusters are fenced.'),
+    message: (
+      <Trans t={t}>
+        <ul>
+          <li>
+            Check the fencing status for your primary and target cluster. Both
+            clusters should be unfenced for initiating relocation.
+          </li>
+          <li>
+            To unfence your cluster, refer to the documentation
+            <ViewDocumentation doclink={DOC_LINKS.UNFENCING} />.
+          </li>
+        </ul>
+      </Trans>
+    ),
+    variant: AlertVariant.danger,
+  },
+  [ErrorMessageType.PRIMARY_CLUSTER_IS_NOT_FENCED]: {
+    title: t('Primary cluster is unfenced.'),
+    message: (
+      <Trans t={t}>
+        <ul>
+          <li>
+            The status for your primary cluster must be fenced for initiating
+            failover. Check the status and try again.
+          </li>
+          <li>
+            To fence your cluster, follow the instructions in the documentation
+            <ViewDocumentation doclink={DOC_LINKS.FENCING} />.
+          </li>
+        </ul>
+      </Trans>
+    ),
+    variant: AlertVariant.danger,
+  },
+  [ErrorMessageType.TARGET_CLUSTER_IS_FENCED]: {
+    title: t('Target cluster is fenced.'),
+    message: (
+      <Trans t={t}>
+        <ul>
+          <li>
+            The status for your target cluster must be unfenced for initiating
+            failover. Check the status and try again.
+          </li>
+          <li>
+            To unfence your cluster, follow the instructions in the
+            documentation
+            <ViewDocumentation doclink={DOC_LINKS.UNFENCING} />.
```

Comment:
```suggestion
            <ViewDocumentation doclink={DOC_LINKS.FENCING} />.
```
This is due to latest document change

Reply:
Done